docs/_docs/pages.md

```diff
@@ -59,42 +57,21 @@ and associated URLs might look like:
 └── contact.html  # => http://example.com/contact.html
 ```
 
-### Named folders containing index HTML files
+If you have a lot of pages, you can organize those pages into subfolders. The same subfolders that are used to group your pages in our project's source will exist in the _site folder when your site builds.
 
-There is nothing wrong with the above method. However, some people like to keep
-their URLs free from things like filename extensions. To achieve clean URLs for
-pages using Jekyll, you simply need to create a folder for each top-level page
-you want, and then place an `index.html` file in each page’s folder. This way
-the page URL ends up being the folder name, and the web server will serve up
-the respective `index.html` file. Here's an example of what this structure
-might look like:
+If you want to automatically move pages to the root in the _site directory when your page builds (similar to post behavior), add a permalink in your page's frontmatter like this:
```

Comment:
updated. however, the reason I called out this example is because, as far as I can tell, no permalink configuration made in _config.yml will pull pages out of subfolders and render them in the root directory. 

if you're using jekyll as a blog and just have a few pages, they're probably all in the root directory to begin with anyway. But when you start managing jekyll sites with hundreds of pages, you have to organize the pages in various subfolders. 

to facilitate relative linking so my site can be viewed offline, i want all my pages rendered in the root directory. the only way i've found to flatten them in the root directory on build is by adding the permalink property directly on the page. 

if i had known this earlier from reading the docs, it would have saved me weeks of time. i would like to learn what permalink property can be set in config to achieve this.